traffic_ops/testing/api_contract/v4/conftest.py

```diff
@@ -0,0 +1,103 @@
+"""This module is used to create a Traffic Ops session 
+and to store prerequisite data for endpoints"""
+import json
+import logging
+import sys
+from random import randint
+from urllib.parse import urlparse
+import pytest
+from trafficops.tosession import TOSession
+from trafficops.restapi import OperationError
+
+
+# Create and configure logger
+logger = logging.getLogger()
+
+
+def pytest_addoption(parser):
+    """Passing in Traffic Ops Arguments [Username, Password, Url and Hostname] from Command Line"""
+    parser.addoption(
+        '--to_user', action='store', help='User name for Traffic Ops Session'
+    )
+    parser.addoption(
+        '--to_password', action='store', help='Password for Traffic Ops Session'
+    )
+    parser.addoption(
+        '--to_url', action='store', help='Traffic Ops URL'
+    )
+
+
+@pytest.fixture(name="to_args")
+def to_data(pytestconfig):
```

Comment:
missing argument type and function return type

Reply:
Added argument type and function return type